### PR TITLE
Integration test controller

### PR DESCRIPTION
**Pull request by hakankaradis**
Wip
http://recordit.co/3d5jSFTAuC

### CODE REVIEW

**Comment by gokmen** [comment]
do you guys planning to ship all these test code with the application itself? cc/ @devrim @anonrig

File: client/app/lib/integration-tests/tests/clone_stack_template.coffee [hunk 1]
```diff
@@ -0,0 +1,230 @@
...
+    it """Do you see 'Instructions', 'Credentials' and 'Build Stack' sections?""", -> 
+      assert(false, 'Not Implemented')
+      #assertion here
+
+    it """Do you see 'STACKS', '_Aws Stack_' and 'aws-instance-1' labels on left sidebar??""", -> 
+      assert(false, 'Not Implemented')
+      #assertion here
+
+
+  describe """Click on 'NEXT' and BUILD STACK buttons respectively""", ->
```
Comment (anonrig) [inline client/app/lib/integration-tests/tests/clone_stack_template.coffee]: Why did you use triple quotation marks in here? @hakankaradis

**Comment by anonrig** [comment]
You've added the files that I've written [in this pull request](https://github.com/koding/koding/pull/9551).

Why did you copy the files in my PR to here? @hakankaradis

**Comment by hakankaradis** [comment]
@anonrig I have lost your commits while playing around branches, actually that's why it is `wip`.
I will cherry-pick your commits and merge them with mine and update this pr soon.
Sorry for that!

**Comment by gokmen** [comment]
@hakankaradis can u assign me back once it's ready?

**Pull request closed without merging**